tests/suites/events.js

```diff
@@ -301,7 +301,7 @@ test('setDate: triggers change and changeDate events', function(){
 
     this.dp.setDate(new Date(2011, 2, 5));
 
-    equal(triggered_change, 2);
+    equal(triggered_change, 0);
```

Comment:
Looking at the result of the test, the change event isn't fired anymore on the input field, if we just fire change once it wouldn't be a BC break. The issue is that it fires the change event more then once, if it would just fire once this bug is fixed without the BC break